README.md

```diff
@@ -13,7 +14,12 @@ Gradle is used to handle dependencies.
 - Clone the repo and initialize the [SpongeAPI](https://github.com/SpongePowered/SpongeAPI) dependency: `git clone --recursive https://github.com/LapisBlue/Pore.git`
 - Compile the project using the Gradle wrapper: `./gradlew` (`gradlew` on Windows)
 
+Need Help/Have Questions?
+-------------------------
+
+Having issues using Pore, no problem jump into an IRC client and jump onto #lapis on EsperNet and you'll be sure to get help.
```

Comment:
I have noticed a bit could you point some out, I'm not the best at english.

—
Sent from Mailbox

On Wed, Jan 28, 2015 at 12:18 AM, Maxim Roncace <notifications@github.com>
wrote:

>> @@ -13,7 +14,12 @@ Gradle is used to handle dependencies.
>>  - Clone the repo and initialize the [SpongeAPI](https://github.com/SpongePowered/SpongeAPI) dependency: `git clone --recursive https://github.com/LapisBlue/Pore.git`
>>  - Compile the project using the Gradle wrapper: `./gradlew` (`gradlew` on Windows)
>>  
>> +Need Help/Have Questions?
>> +-------------------------
>> +
>> +Having issues using Pore, no problem jump into an IRC client and jump onto #lapis on EsperNet and you'll be sure to get help.
> Grammar here is a bit weird
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/LapisBlue/Pore/pull/9/files#r23656209